openmrs/apps/clinical/app.json

```diff
@@ -36,6 +36,8 @@
                 { "locale" : "pt", "css": "offline-language-portuguese-brazil"}
             ]
         },
+        "ignoredTabularViewHeadings": ["display", "uuid", "image", "activeVisitUuid", "hasBeenAdmitted", "programUuid", "enrollment"],
```

Comment:
| OT Screen | Radiology Screen | Orders Screen | Bed Management Screen |
|-----------------------|-----------------------|-----------------------|-----------------------|
| ![Screenshot 2024-04-23 at 9 38 16 PM](https://github.com/Bahmni/default-config/assets/121226043/602bdef2-6352-4ff8-bfd8-1152d5195937) | ![Screenshot 2024-04-23 at 9 39 50 PM](https://github.com/Bahmni/default-config/assets/121226043/9f0f9ce7-4126-49ad-855f-ad5e0756f42d) | ![Screenshot 2024-04-23 at 9 43 40 PM](https://github.com/Bahmni/default-config/assets/121226043/a36dbf66-3f26-4d16-b6a6-4b19f64c80fc) | ![Screenshot 2024-04-23 at 9 47 02 PM](https://github.com/Bahmni/default-config/assets/121226043/0038e0d7-b5d1-4615-ba9b-8c6aab8b47ae) |

The sort logic for patient search is used across Bahmni on screens that support patient search. The following configuration is to be added to these screens to specify which columns to ignore or how to identify the identifier heading. 

Note that some values in the arrays below may not be relevant to the clinical/app settings and should be added to the other app.json file.